From f6976aaccc195fb298e1c61d20d5b75ea5ac52f5 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 18 Sep 2020 16:09:49 -0500
Subject: [PATCH 1/4] Exclude dotnet-suggest from source-build.

This is not used by any downstream repos in source-build and adds a
bunch of prebuilts, so we can leave this out for now.
---
 .../dotnet-suggest.csproj                     |   1 +
 1 files changed, 1 insertion(+)

diff --git a/src/System.CommandLine.Suggest/dotnet-suggest.csproj b/src/System.CommandLine.Suggest/dotnet-suggest.csproj
index f529484..ef9c3bb 100644
--- a/src/System.CommandLine.Suggest/dotnet-suggest.csproj
+++ b/src/System.CommandLine.Suggest/dotnet-suggest.csproj
@@ -13,6 +13,7 @@
     <DotnetSuggestBuildNumber Condition="'$(VersionSuffixDateStamp)' != ''">.$(VersionSuffixDateStamp).$(VersionSuffixBuildOfTheDay)</DotnetSuggestBuildNumber>
     <VersionPrefix>1.1$(DotnetSuggestBuildNumber)</VersionPrefix>
     <VersionSuffix></VersionSuffix>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
   </PropertyGroup>
 
   <PropertyGroup>
-- 
2.18.0

